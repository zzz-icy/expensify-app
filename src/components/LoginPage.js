import React from 'react';


const LoginPage = () => (
    <div>
        <p>This is my LoginPage component!</p>
        <button>Login</button>
    </div>
);
export default LoginPage;