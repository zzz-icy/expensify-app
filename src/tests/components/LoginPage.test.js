// react-test-renderer
import React from 'react';
// import ReactShallowRenderer from 'react-test-renderer/shallow'; // ES6
import { shallow } from 'enzyme';
import LoginPage from '../../components/LoginPage'; // we just want the unconnected version
import "../setupTests.js";
import toJSON from 'enzyme-to-json';

test('should render LoginPage properly', () => {
    const wrapper = shallow(<LoginPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();

});
