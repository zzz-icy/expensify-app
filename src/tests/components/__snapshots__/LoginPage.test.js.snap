// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoginPage properly 1`] = `
<div>
  <p>
    This is my LoginPage component!
  </p>
  <button>
    Login
  </button>
</div>
`;
